mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeans.scala

```diff
@@ -238,7 +241,8 @@ class KMeans private (
   /**
    * Implementation of K-Means algorithm.
    */
-  private def runAlgorithm(data: RDD[VectorWithNorm]): KMeansModel = {
+  private def runAlgorithm(data: RDD[VectorWithNorm],
```

Comment:
Please follow the Spark style guide: [https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide].
Here, for multi-line method headers, put 1 arg per line, and put the initial arg on the line below the method name.  Check out surrounding code for examples.